fastplotlib/widgets/image.py

```diff
@@ -3,28 +3,35 @@
 
 import numpy as np
 
-
 from ..layouts import GridPlot
 from ..graphics import ImageGraphic
 from ..utils import calculate_gridshape
 from .histogram_lut import HistogramLUT
 from ..layouts._utils import CANVAS_OPTIONS_AVAILABLE
 
-
 if CANVAS_OPTIONS_AVAILABLE["jupyter"]:
     from ..layouts._frame._ipywidget_toolbar import IpywidgetImageWidgetToolbar
 
 if CANVAS_OPTIONS_AVAILABLE["qt"]:
     from ..layouts._frame._qt_toolbar import QToolbarImageWidget
 
+# How many dimensions to display a single image
+# If key is 0 this means no RGB (so each image is 2 dims). If key is 1, it's 3
+RGB_DIM_MAP = {0: 2, 1: 3}
```

Comment:
`IMAGE_DIM_COUNTS = {"gray": 2, "rgb": 3}`

and then map `rgb: list[bool]` to `IMAGE_DIM_COUNTS`